graphql-kotlin-schema-generator/src/test/kotlin/com/expediagroup/graphql/generator/types/InterfaceBuilderTest.kt

```diff
@@ -67,17 +64,4 @@ internal class InterfaceBuilderTest : TypeTestHelper() {
         assertEquals(1, result?.directives?.size)
         assertEquals("simpleDirective", result?.directives?.first()?.name)
     }
-
-    @Test
-    fun `verify interface is build only once`() {
```

Comment:
removal of this test (and similar ones for union and objects) implies we no longer cache them?

Reply:
We still cache them but now only in the method `buildIfNotUnderConstruction` this simplifies the logic and makes sure we cache only once